### PR TITLE
APPS-8732 - Update documentation for OpenSearch App Platform Log Forwarding endpoint format.

### PR DESCRIPTION
**Pull request by ElanHasson**
Update documentation for OpenSearch App Platform Log Forwarding endpoint format.

### CODE REVIEW

**Comment by ElanHasson** [comment]
Please merge when approved.

File: specification/resources/apps/models/app_log_destination_open_search_spec.yml [hunk 1]
```diff
@@ -3,9 +3,8 @@ properties:
   endpoint:
     type: string
     description: >-
-      OpenSearch API Endpoint. Only HTTPS is supported. Format:
-      <host>:<port>. Default port is 9300.
-    example: "example.com:9300"
+      OpenSearch API Endpoint. Only HTTPS is supported. Format: https://\u003chost\u003e:\u003cport\u003e.
```
Comment (thearyanahmed) [inline specification/resources/apps/models/app_log_destination_open_search_spec.yml:6]: **Question:** Is the unicode characters necessary?

Comment (andrewsomething) [reply specification/resources/apps/models/app_log_destination_open_search_spec.yml:6]: Curious about this as well. Was that intentional? 

```suggestion
      OpenSearch API Endpoint. Only HTTPS is supported. Format: `https://<host>:<port>`.
```

Comment (ElanHasson) [reply specification/resources/apps/models/app_log_destination_open_search_spec.yml:6]: This is what is generated by our tooling.

**Review by andrewsomething — APPROVED** [review]
:+1:

**Pull request merged**